app/controllers/container_topology_controller.rb

```diff
@@ -22,6 +22,11 @@ def data
     render :json => {:data => generate_topology(params[:id])}
   end
 
+  def refresh_provider
+    render :nothing => true
+    EmsRefresh.queue_refresh([params[:id]])
```

Comment:
@blomquisg @Fryguy, is this the right way to call the background refreshers? I'm trying to mimic here the behavior of hitting the refresh button from provider summary page, but there's some much code there that is irrelevant to here, that I am struggling to understand if indeed it does the work correctly